src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -298,6 +298,11 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "DELETE" }
     ),
+    updateNotificationsLoader: gravityLoader(
+      "me/notifications",
+      {},
+      { method: "PUT" }
```

Comment:
**question:** why put method? (out of curiosity)

Reply:
such HTTP Method is used for [this endpoint](https://github.com/artsy/gravity/blob/main/app/api/v1/me_notifications_endpoint.rb#L58)